lib/config/locales/en.yml

```diff
@@ -60,14 +60,6 @@ en:
           resolution: "Search for attributes that are in the database or set
             the Mongoid.raise_not_found_error configuration option to false,
             which will cause a nil to be returned instead of raising this error."
-        eager_load:
```

Comment:
Everything having to do with this exception must remain in Mongoid for backwards compatibility reasons, even if Mongoid no longer uses this exception.